browser-extension/mv2/src/modules/PreDefinedFunctions.js

```diff
@@ -80,3 +76,25 @@ RQ.registerPredefinedFunction("rq_decrement", {
     return parseInt(num) - parseInt(step);
   },
 });
+
+RQ.registerPredefinedFunction("rq_request_origin", {
```

Comment:
@lazyvab request_origin can be confused with the origin of the request URL. What we want here is to define a predefined function that returns tab_origin or page_origin So should we rename this predefined function to `rq_page_origin`

Reply:
@sachinjain024 The request origin is always defined as the origin which initiated the request. The `origin` header included in a request by the browser also means the same thing.

Also, in case of CORS issue, browser console log looks like:
`Access to fetch at 'https://mysite.com/test-api' from origin 'https://example.com' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://mysite.com' that is not equal to the supplied origin.`

If you think it can still be confusing, can we rename to `rq_request_initiator`?

The reason why I don't want to call it `page_origin` is because if the request is initiated from an iframe, the iframe's origin might be different from page_origin.

Reply:
As discussed, we are renaming it to `rq_origin()`.